Automattic/jetpack
# actions: Don't run Beta Plugin artifact action on forked PRs

#### Changes proposed in this Pull Request:

The existing check that was meant to avoid it was incorrect.
`github.repository` is the repo the PR is created on, not the source
repo, so checking it is mostly pointless. We need to look at
`github.event.pull_request.head.repo.full_name` instead.

#### Jetpack product discussion


None, unfortunately

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* The "Build / Create artifact for Jetpack Beta plugin" job should run on this PR.
* Create a PR from a forked repo based on this PR. The "Build / Create artifact for Jetpack Beta plugin" job should be skipped.
  * For example, PR #21020.
* Merge this. Check that the "Build / Create artifact for Jetpack Beta plugin" job ran on the master merge.
